security/pkg/server/ca/server.go

```diff
@@ -118,6 +118,7 @@ func (s *Server) CreateCertificate(ctx context.Context, request *pb.IstioCertifi
 	var signErr error
 	var cert []byte
 	var respCertChain []string
+	serverCaLog.Debugf("generating a certificate for %v", certOpts.SubjectIDs)
```

Comment:
we have log in L110, i think we cam log more info but add there

Reply:
Sure, I combined this with the existing one at L110. Besides, the level has been changed to Info since it's kind of an important piece of message.